### ISSUE TITLE
SyntaxError with short hand notation `<>` in Astro

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#1562]
- Markuplint version: `4.5.0`
- Parser lang: Astro
- Node.js version: `20.11.0`
- OS: Windows

## Describe the bug

Astro allows the use of `<>` for the short hand of `<Fragment>` 👉[Fragments](https://docs.astro.build/en/basics/astro-syntax/#fragments)

## Code Example

```html
<!-- SyntaxError -->
<>text</>
```

```html
<!-- No error -->
<Fragment>text</Fragment>
```

## The current behavior

```
<markuplint> error: SyntaxError: No tag name: "<" (parse-error)
```

## The expected behavior

No Error
